src/main/java/dk/cngroup/lentils/controller/ClientController.java

```diff
@@ -0,0 +1,42 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.entity.Cypher;
+import dk.cngroup.lentils.entity.Team;
+import dk.cngroup.lentils.service.*;
```

Comment:
Pokud to jde, prosim nepouzivat * wild card v importech. Pokud tam budou nejake services, ktere nepotrebujes, tak se naimportui zbytecne taky. Idealne vzdycky pouzit Ctrl+Alt+O a vic importy neresit ;)